packages/@okta/vuepress-site/docs/guides/saml-inline-hook/main/index.md

```diff
@@ -0,0 +1,338 @@
+  }
+  else {
+
+  console.log("Not a patient. No change to SAML assertion.");
+  response.status(204).send();
+  }
+
+}
+);
+```
+
+## Activate and enable the SAML assertion inline hook
+
+The SAML assertion inline hook must be activated and enabled within your Okta Admin Console.
+
+Activating the SAML assertion inline hook registers the hook with the Okta org and associates it with your external service. Enabling the SAML assertion inline hook associates the hook with your SAML application.
+
+### Activate the SAML assertion inline hook
+
+1. Navigate to the **Workflow** > **Inline Hooks** page.
+
+1. Click **Add Inline Hook** and select **SAML** from the dropdown menu.
+
+1. Add a name for the hook (in this example, "Patient SAML Hook").
+
+1. Add your external service URL, including the endpoint. For example, use your Glitch project name with the endpoint: `https://your-glitch-projectname.glitch.me/SAMLHook`.
+
+1. Include authentication field and secret. In this example:
+
+   * **Authentication field** = `authorization`
+   * **Authentication secret** = `Basic YWRtaW46c3VwZXJzZWNyZXQ=`
+
+1. Click **Save**.
+
+The SAML assertion inline hook is now set up with a status of active.
+
+### Enable the SAML assertion inline hook
+
+1. Navigate to **Applications** and select your SAML application (in this example, "Spring Book SAML").
+
+1. Click the **General** tab.
+
+1. From the **SAML Settings** tile, click **Edit**.
+
+1. Click **Next** to get to the **Configure SAML** tab.
+
+1. From the **SAML Settings** tile, under the **General** heading, click **Show Advanced Settings**.
+
+1. In the **Assertion Inline Hook** field, select your registered inline hook from the dropdown menu (in this example, "Patient SAML Hook").
+
+1. Click **Next** and **Finish**.
+
+The SAML assertion inline hook is now ready for triggering when a user authenticates through the SAML application.
+
+## Preview and test the SAML assertion inline hook
+
+The SAML assertion inline hook is ready for preview and testing. You now have the following applications configured:
+
+* The SAML application (okta-spring-boot-saml-example) is ready to sign in and authenticate users using your Okta org as an IdP.
+* The external service (okta-inlinehook-samlhook) is ready with code to receive and respond to an Okta SAML assertion inline hook call.
+* The Okta org is setup to call the external service when a SAML assertion inline hook is triggered by a user sign-in from the SAML application, and is ready to receive a response.
+
+### Preview the SAML assertion inline hook
+
+1. Navigate to Inline Hooks (**Workflow** > **Inline Hooks**) in your Admin Console.
+
+1. Click on the SAML assertion inline hook name (in this example, "Patient SAML Hook").
+
+1. Click the **Preview** tab.
+
+1. Select a user assigned to your SAML application in the first block titled **Configure Inline Hook request**; that is, a value for the `data.userProfile` field.
```

Comment:
don't have a UI to check, so if it's not a dropdown list, my bad. I'm just trying to avoid the semicolon and make it more simple......
```suggestion
1. In the first block titled **Configure Inline Hook request**, define a value for `data.userProfile` by selecting a user in your org from the **data.userProfile** dropdown list.
```

Reply:
I might add "searching and selecting", although "search" is a field hint. Sorry about the semi-colon, I re-purposed some old copy.